packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -368,6 +369,14 @@ export const preparePortfolioKit = (
           const { portfolioId } = this.state;
           publishStatus(makeFlowStepsPath(portfolioId, id), steps);
         },
+        publishFlowOneStep(
+          id: number,
+          step: number,
+          status: StatusFor['flowStep'],
+        ) {
+          const { portfolioId } = this.state;
+          publishStatus(makeFlowStepPath(portfolioId, id, step), status);
```

Comment:
Rather than a `.flow123.steps.step45` node, how about adding a `.txId` property to `FlowStep` and putting the tx metadata in the `.pendingTxs.tx678` node?

Reply:
I did that now, as discussed.